spec/models/temp_condition_spec.rb

```diff
@@ -0,0 +1,344 @@
+                  start_station_id: 4,
+                  end_station_id: 4,
+                  subscription_type: "Subscriber",
+                  zip_code: "94114")
+
+      Trip.create(duration: 174,
+                  start_date: "2013-08-30",
+                  start_station_name: "Panera",
+                  end_date: "2013-08-29 09:11:00",
+                  end_station_name: "Turing",
+                  bike_id: 288,
+                  start_station_id: 1,
+                  end_station_id: 1,
+                  subscription_type: "Subscriber",
+                  zip_code: "94114")
+
+      Trip.create(duration: 174,
+                  start_date: "2013-08-29",
+                  start_station_name: "Turing",
+                  end_date: "2013-08-29 09:11:00",
+                  end_station_name: "Turing",
+                  bike_id: 288,
+                  start_station_id: 2,
+                  end_station_id: 2,
+                  subscription_type: "Subscriber",
+                  zip_code: "94114")
+
+      Trip.create(duration: 174,
+                  start_date: "2013-08-29",
+                  start_station_name: "Panera",
+                  end_date: "2013-08-29 09:11:00",
+                  end_station_name: "Panera",
+                  bike_id: 288,
+                  start_station_id: 1,
+                  end_station_id: 1,
+                  subscription_type: "Subscriber",
+                  zip_code: "94114")
+
+      Condition.create(date:'2013-08-29',
+                   max_temperature: 70,
+                   mean_temperature: 65,
+                   min_temperature: 60,
+                   mean_humidity: 40,
+                   mean_visibility: 20,
+                   mean_windspeed: 50,
+                   zip_code: 95113,
+                   precipitation:  3)
+
+
+      Condition.create(date:'2013-08-29',
+                   max_temperature: 77,
+                   mean_temperature: 67,
+                   min_temperature: 65,
+                   mean_humidity: 43,
+                   mean_visibility: 22,
+                   mean_windspeed: 30,
+                   zip_code: 95113,
+                   precipitation: 1)
+
+      Condition.create(date:'2013-08-29',
+                  max_temperature: 70,
+                  mean_temperature: 65,
+                  min_temperature: 60,
+                  mean_humidity: 40,
+                  mean_visibility: 20,
+                  mean_windspeed: 50,
+                  zip_code: 95113,
+                  precipitation:  3)
+
+
+      Condition.create(date:'2013-08-30',
+                  max_temperature: 77,
+                  mean_temperature: 67,
+                  min_temperature: 65,
+                  mean_humidity: 43,
+                  mean_visibility: 22,
+                  mean_windspeed: 30,
+                  precipitation:  1,
+                  zip_code: 95113)
+
+      expect(Condition.low_rides_by_weather(70.0)).to eq(3)
+    end
+  end
+end
+
```

Comment:
Great tests. If we have time to refactor, we should put these setups in a class.